### ISSUE TITLE
Aurora doesn't work

### ISSUE THREAD

**Issue by vicdilou** [antonpup/Aurora#1334]
### Actual behavior:

Aurora start up but don't work

### How often does this problem occur?

This problem occurs everytime

### What is the version of Aurora you are experiencing this issue with?

My version is beta v0.6.3-516

### Does the previous version of Aurora have this issue?
Some previous version have this issue.

I have a G910 with the last update of Ghub and the assistant.

**Comment by BrN027** [comment]
I have exactly the same issue. Actually, i've untick start with Windows and i lauch Aurora manually.

**Comment by olokos** [comment]
@vicdilou can you reopen the issue?

**Comment by olokos** [comment]
https://gyazo.com/1986394e1de16932e05ea0b616b8b51c

exactly the same thing with new version

**Comment by simon-wh** [comment]
@olokos Could you post your log file for a launch where Corsair doesn't start up when Aurora does? It should hopefully give us some insight into why it doesn't connect when Aurora starts. I take it Razer connects successfully when Aurora starts?

**Comment by olokos** [comment]
@simon-wh C:\Program Files\Aurora\log.txt is completely empty
Corsair's iCue does start and provides its effects
Razer does connect successfully each time, that's correct.

**Comment by simon-wh** [comment]
@olokos You need to look in %appdata%/Aurora/logs for the log files, or in Aurora go to Settings -> Debug -> Show Logs folder

**Comment by olokos** [comment]
@simon-wh 
Here it is:
https://pastebin.com/3D4A6Xtb

**Comment by simon-wh** [comment]
@olokos From what it looks like, it seems that Aurora is starting up earlier than iCUE so is not able to connect immediately and expends all retries before iCUE starts. 
Could you try adding a delay to the startup task of Aurora?
1. Open Task Scheduler (Either through searching for it in the start menu or WIN+R and type in taskschd.msc and press Enter)
2. Click on "Task Scheduler Library" on the left
3. Find "AuroraStartup"
4. Right click that entry and click Properties
5. Click on the "Triggers" tab
6. Click on the "At log on" entry, then click "Edit" at the bottom of the box
7. Under Advanced Settings check the "Delay task for" and put in a time, you can try experimenting with this, you can type in an exact number, so try maybe 2/3 minutes initially. Then click Ok for everything
The screens should look like below
![image](https://user-images.githubusercontent.com/10797576/51542257-b8e9ea80-1e52-11e9-8d5d-9f15c4f4ace7.png)

Could you let me know how you get on with trying that?

**Comment by olokos** [comment]
That's exactly what I also thought, but I didn't know a way to delay launching of Aurora, back in the days there were Registry keys, msconfig and such. I wasn't really expecting it to be within tasks.

I think the best solution would be to implement some sort of check if corsair sdk service is running before launching aurora corsair profile.

It does work perfectly fine with this solution though, thanks! 😄

**Comment by simon-wh** [comment]
@olokos Yeah, I'm planning to do checks and stuff, but I also think what probably didn't help was that it only had 3 retry attempts set, so it wasn't trying very hard get it setup. I think I'll probably set it to do more retries with a bigger gap between them which should hopefully help with other cases.

**Comment by olokos** [comment]
That sounds even better than what I came up with! My suggestion would be 5 checks with 1 minute delay inbetween :D
